src/lib/route.js

```diff
@@ -135,11 +158,21 @@ class Route {
       points: this.getPoints(),
       min_message_window: this.minMessageWindow,
       source_account: this.sourceAccount,
+
       destination_precision: this.destinationPrecision,
-      destination_scale: this.destinationScale
+      destination_scale: this.destinationScale,
+
+      added_during_epoch: this.addedDuringEpoch
```

Comment:
It's needed internally, and needs to be returned in the json form, for the route-broadcaster to have easy access to it, but it's removed before the schema is validated